packages/graphql-explorer/src/ObjectSection.tsx

```diff
@@ -123,11 +124,16 @@ function ObjectSectionField({
   );
 }
 
+interface ObjectSectionProps extends SectionProps<Obj, g.GraphQLObjectType> {
+  extraFields?: ListGroupItem | ListGroupItem[];
```

Comment:
nope, unfortunately you can't do that. when you do `<ListGroupItem ... />` this returns a ReactNode, not a ListGroupItem. I am pretty sure that if you apply this change to lando-ui, ts will complain. in general this is a broad limitation of the react type system. everything is a node, or at best an element, but you cannot discern what type of element the element is.

Reply:
You're right. I noticed this when testing as you suspected. I decided not to use this though for now as it intermingles custom fields with the schema fields. Better to separate them in another component like it was before I think